kf_lib_data_ingest/target_api_plugins/kids_first_dataservice.py

```diff
@@ -98,19 +98,27 @@ class Study:
 
     @classmethod
     def get_key_components(cls, record, get_target_id_from_record):
-        return {
-            "data_access_authority": not_none(record[CONCEPT.STUDY.AUTHORITY]),
-            "external_id": not_none(record[CONCEPT.STUDY.ID]),
-        }
+        kfid = record.get(cls.target_id_concept) or record.get(
```

Comment:
Since an unregistered project ID is not necessarily a KF ID, we may rename this variable?

Reply:
it's being assigned to the kf_id field. should it not?